CONTRIBUTING.md

```diff
@@ -1,39 +1,97 @@
+
+We track Kubescape issues and bugs on the [project board](https://github.com/orgs/kubescape/projects/4)
+
+Contributors should pay attention to three types of labels:
+1. [good first issue](https://github.com/orgs/kubescape/projects/4/views/3?sliceBy%5Bvalue%5D=good+first+issue) - these are issues suitable for new contributors.
+2. [open for contribution](https://github.com/orgs/kubescape/projects/4/views/4?sliceBy%5Bvalue%5D=open+for+contribution) - these are issues suitable for someone who isn't a core maintainer.
+3. [help wanted](https://github.com/orgs/kubescape/projects/4/views/5?sliceBy%5Bvalue%5D=help+wanted) - these are issues that require knowledge beyond core Kubescape functionality.
+
+Once you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it and assign yourself.
+
+## Working on issues
+We categorize contributions into two:
+* Small code changes or fixes, whose scope is limited to documentation, minor fixes, development that involves no more than a file or two.
+* Complex features and improvements, with potentially unlimited scope
+
+If you are working on a small change, feel free to open a Pull Request.
+
+If you want to work on a bigger change, please discuss the change you are plannning with the community and the maintainers. In this case, opening an issue in the  applicable repository and raising the improvement you want to add in the kubescape-dev slack or community meeting is a great start!
+
+Getting sign-off before embarking on a big change is important so the maintainers can help guide you in the right direction. 
+
+## Ask for help
+
+The best way to reach us with a question when contributing is to ask on:
+
+* The original github issue
+* The kubescape-dev Slack channel
 
 ## Build and test locally
 
 Please follow the [instructions here](https://github.com/kubescape/kubescape/wiki/Building).
 
-## Pull Request Process
+## Pull Request process
 
-1. Ensure any install or build dependencies are removed before the end of the layer when doing a
-   build.
-2. Update the README.md with details of changes to the interface, this includes new environment
-   variables, exposed ports, useful file locations and container parameters.
+1. Ensure any install or build dependencies are removed before the end of the layer when running a  build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
 3. Open Pull Request to the `master`/`main` branch.
 4. We will merge the Pull Request once you have the sign-off.
 
-## Developer Certificate of Origin
+<!-- Ben/Rotem/Matthias the section below was in the CNCF template and seems important to me. WDYT?
+
+## Development Environment Setup
+
+[Instructions](https://contribute.cncf.io/maintainers/github/templates/required/contributing/#development-environment-setup)
+
+⚠️ **Explain how to set up a development environment**-->
```

Comment:
I don't think this is relevant here, it depends on each repo with their own environment setup